test/cql-pytest/util.py

```diff
@@ -63,15 +63,38 @@ def format_tuples(tuples=None, **kwargs):
     body = ', '.join(f"'{key}': '{value}'" for key, value in tuples.items())
     return f'{{ {body} }}'
 
+def keyspace_has_tablets(cql, keyspace):
+    """ Return true if the test_keyspace was created with tablets.
+
+    We support running cql-pytest against an older version of scylla, so we do
+    the detection in a way that accounts for scylla possibly not even knowing
+    what tablets is.
+
+    If the keyspace was created with tablets, it will have an entry in
+    `system_schema.scylla_keyspaces`, with `initial_tablets` set to 0.
```

Comment:
As the code below does, you only need to check if it exists - it doesn't necessarily need to be 0 (even though 0 is indeed the default, maybe it was chosen explicitly).

Reply:
Yes, will fix.

Reply:
Fixed in v2.